tests/format/angular/angular/attr-value-hugging.component.html

```diff
@@ -0,0 +1,10 @@
+<div
+  [target]=" 'string literal' "
+  [target]=" `template literal` "
+  [target]=" `template
+literal` "
+  *directive=" 'string literal' "
+  *directive=" `template literal` "
+  *directive=" `template
+literal` "
+></div>
```

Comment:
Too many options combinations in this dir, and useless, let's put in another dir only test with default options, and make the string longer, just like #7479